go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubernetes-csi/external-snapshotter/v6
 
-go 1.17
+go 1.19
```

Comment:
Why this version bump?

It causes the unit test to fail (still runs with Go 1.18, which doesn't support `go 1.19`) and shouldn't be necessary (go 1.19 can compile code that only depends on features from `go 1.17`).